autoload/go/doc.vim

```diff
@@ -60,12 +56,11 @@ endfunction
 function! go#doc#Open(newmode, mode, ...) abort
   " With argument: run "godoc [arg]".
   if len(a:000)
-    let bin_path = go#path#CheckBinPath('godoc')
-    if empty(bin_path)
+    if empty(go#path#CheckBinPath(g:go_doc_command[0]))
       return
     endif
 
-    let command = printf("%s %s", go#util#Shellescape(bin_path), join(a:000, ' '))
+    let command = printf("%s %s", join(g:go_doc_command, ' '), join(a:000, ' '))
```

Comment:
This removes the `go#util#Shellescape()` part. You should probably map that to every entry in `g:go_doc_command`?

(This should probably be done by the `System()` function, but changing that would require careful testing of all the `System()` invocations and may break people's configs).